### DISCUSSION TITLE
Add config option for which Decrees should spawn on the board

### DISCUSSION THREAD

**Discussion by NiveStinks** [ejektaflex/Bountiful#256] [Mod Ideas]
Are all decrees eligible to spawn on bounty boards from the start? I have plans to make certain decrees gated by exploration or expensive trades and I don't want bounty boards to randomly have them by default.

**Comment by ejektaflex** [comment]
Hi! Yes, they're all equally eligible. Given your use case, there's definitely an argument to be made about restricting which Decrees can spawn on the board via the config!

I'll look into implementing this in a future version for 1.20.2+.

**Comment by NiveStinks** [comment]
Oof, no 1.20.1? Bummer! Then again, not all the mods I'm waiting for are even ported to 1.20.anything so maybe waiting for this feature in 1.20.2 won't be too terrible.

**Reply by ejektaflex** [reply]
I'm in the same boat, waiting for Botania 🥲 we'll see what happens!

**Comment by ejektaflex** [comment]
Hey @NiveStinks , I've added this feature via this commit:

https://github.com/ejektaflex/Bountiful/commit/a6b9544589069db0f7a3998f3ecdf28690dc33fa

It should show up in the next version of Bountiful (most likely for 1.20.4). Thanks for your patience, and for the great idea.

**Comment by NiveStinks** [comment]
I've recently regained interest in my Minecraft modpack and I was wondering if you have any plans to backport this change to version 1.20.1? I figured by now the mods I wanted bumped up to 1.20.4 would have by now but sadly they haven't been.

**Comment by Numilani** [comment]
I would also be interested in seeing this backported - 1.20.1 seems to be the version many mods have "stabilized to" (including some that have explicitly stated they won't go further), and this particular feature seems rather important for preventing some odd behaviors.

**Comment by ejektaflex** [comment]
I don't have the time to do a backport - I'm already a year behind on the 1.21 release, sorry! You can look forward to this being present in all future releases.
